config/grafana/dashboards/integration-service-dashboard.json

```diff
@@ -886,101 +886,6 @@
       "yBucketNumber": null,
       "yBucketSize": null
     },
-    {
```

Comment:
> gridPos

Hi Hongwei! I've adjusted the code accordingly and also slightly adjust the visual lineup to prevent the panels from stacking as they were. Have a look if you get the chance to make sure it's all ok with you!

Reply:
Thank you very much for updating all of them!